core/ledger/confighistory/mgr.go

```diff
@@ -220,6 +220,9 @@ func (r *Retriever) ExportConfigHistory(dir string, newHashFunc snapshot.NewHash
 		return nil, err
 	}
 	metadataHash, err := metadataFileWriter.Done()
+	if err != nil {
```

Comment:
Hmm, what am I missing here, why is it a good thing to ignore that error?

Reply:
Not sure what you mean. Looks like the typical error checking pattern

```go
if err != nil {
    return nil, err
}
```

There was no error check before and now there is.

Reply:
Sorry, my bad, I checked out the changes, used meld to inspect the diff but inverted the branches so in that particular file it looked like you removed the error check :facepalm: